pipeline/coordinator.go

```diff
@@ -361,9 +363,32 @@ func (c *Coordinator) finishJob(job *JobInfo, out *HandlerOutput, err error) {
 		WithLabelValues(labels...).
 		Add(float64(job.transcodedSegments))
 
+	postgresMetrics(job)
```

Comment:
We should probably run this as a goroutine so that it's non-blocking

Reply:
yep definitely, i was still in the POC mindset :)